HERACLES-3/HERACLES-4: Implement ECC and Phone Call form feedback

acrowthe opened the pull request.

acrowthe committed:
HERACLES-3/HERACLES-4: Implement ECC and Phone Call form feedback

sashaandjic commented:
Why is the Event Cardiac's `ec_artery_percentage` marked as `INVALID` here:
![image](https://github.com/data-team-uhn/cards/assets/23243244/7e606d67-21a2-4aec-aad9-59ac4c84df65)
![image](https://github.com/data-team-uhn/cards/assets/23243244/48ae3274-64ec-425e-9303-ea50b1b1422b)
![image](https://github.com/data-team-uhn/cards/assets/23243244/9159673d-3ad8-4965-ab93-666353f686a2)

marta- commented:
> Found [bug ](https://phenotips.atlassian.net/browse/CARDS-2475)unrelated to this PR that is causing, the Event Cardiac's `ec_artery_percentage` be marked as `INVALID`

There's a workaround for now: setting maxAnswers to 2 (or 0, or anything greater than 1) on range questions.

marta- commented on heracles-resources/clinical-data/src/main/resources/SLING-INF/content/Questionnaires/Event - Clinical Cardiac.xml:
```diff
@@ -9390,427 +8512,3377 @@
+					<primaryNodeType>cards:Question</primaryNodeType>
+					<property>
+						<name>maxAnswers</name>
+						<value>1</value>
```
```suggestion
						<value>2</value>
```

marta- committed:
HERACLES-3: Event - Clinical Cardiac form

Update maxAnswers on range question to prevent the answer being marked as INVALID

sashaandjic commented:
The decimal date range problem is fixed.

sashaandjic reviewed (approved):
problem fixed

marta- commented:
Closing outdated PR.
